addons/microdagr/functions/fnc_updateDisplay.sqf

```diff
@@ -89,7 +89,7 @@ case (APP_MODE_INFODISPLAY): {
                     ([_bearing, 3, 1] call CBA_fnc_formatNumber) + "°" //degree symbol is in UTF-8
                 };
                 private _2dDistanceKm = ((getPosASL ACE_player) distance2D _targetPosLocationASL) / 1000;
-                _rangeText = format ["%1km", ([_2dDistanceKm, 1, 1] call CBA_fnc_formatNumber)];
+                _rangeText = format ["%1km", ([_2dDistanceKm, 1, 3] call CBA_fnc_formatNumber)];
```

Comment:
`CBA_fnc_formatNumber` can be replaced by the `toFixed` command 👍

Reply:
done